.travis.yml

```diff
@@ -14,6 +14,8 @@ notifications:
     on_start: never # options: [always|never|change] default: always
 after_success:
   - npm run semantic-release
+before_script:
+  - yarn run snyk auth 3c19c1a7-d306-46f3-867c-525d822ab04d
```

Comment:
remove this key, ill add it to travis

Reply:
now SNYK_TOKEN env var, should work by default. No need to authhttps://blog.travis-ci.com/2017-04-20-continuous-security-snyk-travis-ci/

Reply:
hopefully nobody spams my snyk account now lol

Reply:
lol hope not :)